soc/riscv/riscv-ite/common/chip_chipregs.h

```diff
@@ -866,6 +866,10 @@
 #define IT8XXX2_WUC_WUBEMR1 (IT8XXX2_WUC_BASE + 0x3c)
 #define IT8XXX2_WUC_WUBEMR5 (IT8XXX2_WUC_BASE + 0x0f)
 
+#define IT8XXX2_WUC_WUEMR3 ECREG(IT8XXX2_WUC_BASE + 0x02)
+#define IT8XXX2_WUC_WUESR3 ECREG(IT8XXX2_WUC_BASE + 0x06)
+#define IT8XXX2_WUC_WUENR3 ECREG(IT8XXX2_WUC_BASE + 0x0a)
```

Comment:
These should be sorted numerically by the register offset.  Also, you need to be consistent with the macro definitions.  For example you have IT8XXX2_WUC_WUEMR1 defined as an plain 32-bit value, while the new definitions are cast as ECREG types.

A quick scan through this file shows that all the other registers are defined using ECREGn macros.  Please fix the existing Wake-Up control register definitions to match.